_posts/2018-01-02-this-is-not-a-test.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "This Is Not a Test"
+date:   2018-01-02 12:00:00 -0400
+categories: culture interviews
+author: Janice Smith
+---
+
+At forward financing how we work is something we are always trying to improve on.
```

Comment:
Capitalize Forward Financing. Rewrite in active voice:

"At Forward Financing, we are always trying to improve how we work."

Reply:
💯